go/pkg/tool/tool.go

```diff
@@ -656,6 +656,7 @@ func (c *Client) ExecuteAction(ctx context.Context, actionDigest, actionRoot, ou
 	case command.LocalErrorResultStatus:
 		oe.WriteErr([]byte(fmt.Sprintf("Local error: %v.\n", ec.Result.Err)))
 	}
+	fmt.Printf("Result: %+v\n", ec.Result)
```

Comment:
Is this intended or a leftover from testing?

Reply:
This is intended because it does not show me if the action is `cache hit` or `remote execution` or `remote error` in the terminal when I run the action on another RBE instance. I think we should printout the result here.

BTW, I eventually figure out, by design, this remotetool does not use cache hit, it always remote execute a action without cache hit.